server/controllers/TraceController.ts

```diff
@@ -21,52 +21,84 @@ export class TraceController {
-        TraceModel.getIndividualTraceView(req, res, next);
+        TraceModel.getSearchBarTraceView(req, res, next);
       }
       catch(err) {
         return next({
           log: `Error in TraceController.getCluster: ${err}`,
           status: 500,
           message: {
-            err: 'Error occured while retrieving Trace View Data in TraceController',
+            err: 'Error occured while retrieving search bar trace data in TraceController',
           },
         });
       }
     }
-    public static getPodDetails(req: Request, res: Response, next: NextFunction) {
+    
+    public static getTraceViewServices(req: Request, res: Response, next: NextFunction) {
       try{
         console.log("In Tracecontroller");
-        TraceModel.getIndividualPodData(req, res, next);
+        TraceModel.getTraceViewServices(req, res, next);
       }
       catch(err) {
         return next({
           log: `Error in TraceController.getCluster: ${err}`,
           status: 500,
           message: {
-            err: 'Error occured while retrieving Trace Pod Data in TraceController',
+            err: 'Error occured while retrieving trace view services data in TraceController',
+          },
+        });
+      }
+    }
+    
+
+    public static getTraceViewData(req: Request, res: Response, next: NextFunction) {
+      try{
+        console.log("In Tracecontroller");
+        TraceModel.getIndividualTraceView(req, res, next);
+      }
+      catch(err) {
+        return next({
+          log: `Error in TraceController.getCluster: ${err}`,
+          status: 500,
+          message: {
+            err: 'Error occured while retrieving Trace View Data in TraceController',
           },
         });
       }
     }
-    public static getSpanDetails(req: Request, res: Response, next: NextFunction) {
+    public static getPodDetails(req: Request, res: Response, next: NextFunction) {
       try{
         console.log("In Tracecontroller");
-        TraceModel.getIndivSpanDetails(req, res, next);
+        TraceModel.getIndividualPodData(req, res, next);
       }
       catch(err) {
         return next({
           log: `Error in TraceController.getCluster: ${err}`,
           status: 500,
           message: {
-            err: 'Error occured while retrieving Span Details Data in TraceController',
+            err: 'Error occured while retrieving Trace Pod Data in TraceController',
           },
         });
       }
     }
+    // public static getSpanDetails(req: Request, res: Response, next: NextFunction) {
+    //   try{
+    //     console.log("In Tracecontroller");
+    //     TraceModel.getIndivSpanDetails(req, res, next);
+    //   }
+    //   catch(err) {
+    //     return next({
+    //       log: `Error in TraceController.getCluster: ${err}`,
+    //       status: 500,
+    //       message: {
+    //         err: 'Error occured while retrieving Span Details Data in TraceController',
+    //       },
+    //     });
+    //   }
+    // }
```

Comment:
A lot of commented out code here. Should we be keeping this in the codebase?

Reply:
we need to keepthis forrichard.